lib/public/views/Logs/Create/TemplatedLogCreationModel.js

```diff
@@ -49,10 +50,14 @@ export class TemplatedLogCreationModel extends LogCreationModel {
      *
      * @param {function} [onCreation] function called when log is created, with the id of the created log
      * @param {LogCreationRelations} relations the relations of the log
+     * @param {LogCreationTemplateData} templateData the template data of the log
      */
-    constructor(onCreation, relations) {
+    constructor(onCreation, relations, templateData) {
```

Comment:
I think you should extract templateKey from templateData as a different parameter

Reply:
done